### ISSUE TITLE
Canon EOS Kiss X10: Wifi not connecting?

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#952]
The camera in https://github.com/kinu-garage/hut_10sqft/issues/692, I remember months ago I was able to connect it to wifi (and automatically upload to my Google Drive), but today I'm seeing:
```
Connect device to camera using network settings below
SSID: xxxxxxxx
```
I'm waiting for minutes but no progress.

**Comment by 130s** [comment]
I _think_ I typed in the correct the password but the page not moving forward for minutes. This was from "Menu" physical button on top-left on the device panel.

Then from video monitoring page on the device panel, somehow I was asked to choose where to upload where I saw the folder name on my G-Drive or the cellphone name of mine (I must've somehow linked the camera and my phone before), then chose G-Drive and typed the password then the camera's panel shows wifi icon so I assume it's now connected.

Ok now on router's connected device page shows the camera's Mac address. On the camera's panel I had to explicitly choose image(s) --> Click "Set" on the panel --> Click "OK" (to end "set"ting) --> Panel asks something like "send" (I forgot the exact phrase) then finally showed progress page.

Very unintuitive but it's probably ok.
